src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -10,94 +10,180 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
 
 ## The teams
 
-On GitHub, there are mainly three teams working together on the project.
+On GitHub, there are mainly three teams (employees of PrestaShop's company) working together on the project.
 
-- Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers and Wording Managers.
+- Core Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers, and a Product Content Manager.
 - Quality Assurance (QA) team: this team takes care of ensuring the quality of the software. It has QA analysts and QA automation engineers.
-- Developers team a.k.a Core Team: this team takes care of the code and act as maintainer for the project.
+- Core Developers Team: this team takes care of the code and act as maintainer for the project.
```

Comment:
Technically the Code Developers Team is the same as the Maintainers team.

I'd remove the "employees of PrestaShop's company" note above and replace it with a note stating that at the time of writing, most of the team's members are employees of the PrestaShop Company, but not all of them.

Reply:
There's also the question of project governance and the ambiguous roles of QA and Product Teams...

Reply:
> Technically the Core Developers Team is the same as the Maintainers team.

Yes, but I talk here of the teams working for PS company and who work on Github

> There's also the question of project governance and the ambiguous roles of QA and Product Teams

Yes, that's another big question that we are studying

Reply:
See https://github.com/PrestaShop/docs/pull/807/commits/7875579b171ffd8d381c7a41bee1aa8565dd75ff